service_management/azure/lib/azure/virtual_machine_management/serialization.rb

```diff
@@ -99,6 +99,10 @@ def self.role_to_xml(params, image, options)
                     options[:tcp_endpoints],
                     options[:existing_ports]
                   ) if options[:tcp_endpoints]
+                  udp_endpoints_to_xml(
+                      xml,
+                      options[:udp_endpoints]
```

Comment:
should we also pass on options[:existing_ports] to udp_endpoints_to_xml method like we do in tcp_endpoints_to_xml?